src/fparser/two/Fortran2003.py

```diff
@@ -2935,6 +2935,13 @@ def tostr(self):
             s += ' ' + str(self.items[3])
         return s
 
+    def get_name(self):
+        '''Provides the entity name as an instance of the `Name` class.
+
+        :rtype: `Name`
```

Comment:
Please could you use Sphinx auto-api markup for this type, i.e. ":py:class:`fparser.two.Fortran2003.Name`" (without the double quotes).

Reply:
Sure, will do! [`default_role`](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-default_role) might be a useful config for Sphinx